src/app/card/card.components.ts

```diff
@@ -0,0 +1,51 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { Diary } from '../model/diary';
+
+@Component({
+  selector: 'app-card',
+  templateUrl: './card.component.html',
+  styleUrls: ['./card.component.scss'],
+})
+export class CardComponent implements OnInit {
+  @Input() id: string = '';
+  @Input() name: string = 'Anonymous';
+  @Input() description: string = '';
+  @Input() title: string = '';
+
+  constructor() {}
+
+  seeMore: boolean = false;
```

Comment:
variable declaration should go to the top. Always place constructor, life cycle hooks after variable declaration

Reply:
done